.github /dependabot.yml

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Looks like there is a space in the file path. Also, we don't need the license header

Reply:
Great catch!